source/features/quick-reactions.tsx

```diff
@@ -0,0 +1,98 @@
+import delegate, {DelegateEvent} from 'delegate-it';
+import React from 'dom-chef';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager.js';
+
+const emojiSortOrder = ['THUMBS_UP', 'THUMBS_DOWN', 'LAUGH', 'HOORAY', 'CONFUSED', 'HEART', 'ROCKET', 'EYES'];
+
+function optimisticReactionFromPost(event: DelegateEvent<MouseEvent>): void {
+	const reaction = event.delegateTarget as HTMLButtonElement;
+	updateReaction(reaction);
+}
+
+function optimisticReactionFromMenu(event: DelegateEvent<MouseEvent>): void {
+	const reactionButton = event.delegateTarget as HTMLButtonElement;
+	const reactionsMenu = reactionButton.closest('reactions-menu')!;
+	const details = reactionsMenu.querySelector('details');
```

Comment:
Can you replace every querySelector usage with `$` from `select-dom`? The only exception is when the line has multiple traversing, e.g. `x.closest().querySelector('a')` is still more readable than `$('a', x.closest())`